docs/components/concepts/workflow-patterns.md

```diff
@@ -0,0 +1,296 @@
+
+:::note
+The escalation event is supported in Camunda Platform 7, but not yet in Camunda Platform 8. It is on the roadmap and will eventually be available in Camunda Platform 8.
+:::
+
+<div bpmn="workflow-patterns/escalation.bpmn" callouts="event" />
+
+<span className="callout">1</span>
+
+An escalation event can be thrown from any of the called subprocesses and is picked up by its parent to start something in parallel, as this is a non-interrupting event (dashed line).
+
+The subprocess can raise the escalation any time:
+
+<div bpmn="workflow-patterns/escalation-sub.bpmn" callouts="event" />
+
+### Broadcasts and engine-wide events
+
+While messages are always targeted at one specific process instance, you might also want to inform many processes about an event at once. For example, you might regularly adjust certain customer scoring rules that always should be taken into account immediately. This can be implemented using the [signal event](/docs/components/modeler/bpmn/bpmn-coverage/).
+
+:::note
+The escalation event is supported in Camunda Platform 7, but not yet in Camunda Platform 8. It is on the roadmap and will eventually be available in Camunda Platform 8.
+:::
+
+<div bpmn="workflow-patterns/signal-catch.bpmn" callouts="signal"/>
+
+<span className="callout">1</span>
+
+The signal event is caught and in this case interrupts the onboarding to go back to score the customer again.
+
+## Handling errors
+
+Handling exceptions well is one of the most important capabilities of a workflow engine, and it needs built-in support from the modeling language.
+
+You might also want to look into our [best practice: modeling beyond the happy path](/docs/components/best-practices/modeling/modeling-beyond-the-happy-path/) to understand possibilities.
+
+### Error scopes
+
+The reaction to errors might need to be different depending on the current state of the process. This can be achieved by using [subprocesses](/docs/components/modeler/bpmn/embedded-subprocesses/) in combination with either [boundary events](/docs/components/modeler/bpmn/events/#boundary-events) or [event subprocesses](/docs/components/modeler/bpmn/event-subprocesses/).
+
+<div bpmn="workflow-patterns/subprocess-error.bpmn" callouts="errorEvent, errorEventSubprocess" />
+
+<span className="callout">1</span>
+
+This error event only catches errors from the subprocess "clearing". The idea here would be that in case of any clearing service not being available, the order is assumed cleared. Note that this example is mainly built for illustration, and does not necessarily mean this is the best way to solve this business requirement.
+
+<span className="callout">2</span>
+
+Alternatively, this error event subprocess is triggered whenever there is a fraud detected, independent where this error is raised.
```

Comment:
What do you mean by "independent where this error is raised"? Outside of where this error is raised?

Reply:
It's not attached like a boundary event, it listens throughout the process, including other subprocesses. Maybe "independent of whether the error occurs in any subprocess or parent process"?

Reply:
Like this: https://github.com/camunda/camunda-platform-docs/pull/1006/commits/9aee9f96c83fa0247aa3e52506470ad1562951a3

Reply:
Excellent!